blast/templates/blast/results.html

```diff
@@ -3,7 +3,6 @@
 {% load pipeline %}
 
 {% block navbar%}
-<li><a href="./{{task_id}}">Results</a></li>
```

Comment:
What's this?

Reply:
Does not appear to be visible in production. This is overwritten with the new navbar.